profiles/patriciadiaz.json

```
@@ -0,0 +1,6 @@
+{
+    "image": "https://user-images.githubusercontent.com/15069545/173307507-7917cca8-4dba-465b-bb3c-321d82cb5997.png",
+    "issueId": 1168,
+    "name": "Patricia",
+    "username": "devpatricia"
```

The indentation seems to be wrong here.
```suggestion
  "image": "https://user-images.githubusercontent.com/15069545/173307507-7917cca8-4dba-465b-bb3c-321d82cb5997.png",
  "issueId": 1168,
  "name": "Patricia",
  "username": "devpatricia"
```